src/seedu/addressbook/data/person/Address.java

```diff
@@ -35,25 +31,4 @@ public Address(String address, boolean isPrivate) throws IllegalValueException {
     public static boolean isValidAddress(String test) {
         return test.matches(ADDRESS_VALIDATION_REGEX);
     }
-
-    @Override
-    public String toString() {
-        return value;
-    }
-
-    @Override
-    public boolean equals(Object other) {
```

Comment:
you probably also need to override `equals` for `Address` class so that you can check if the other object is the same type as Address.